taskcluster/docker/linux/Dockerfile

```diff
@@ -86,65 +99,68 @@ RUN locale-gen en_US.UTF-8
 
 # Android SDK
 
-RUN mkdir -p /build/android-sdk
-WORKDIR /build
+RUN mkdir -p /builds/worker/android-sdk
+WORKDIR /builds/worker
 
-ENV ANDROID_HOME /build/android-sdk
-ENV ANDROID_SDK_HOME /build/android-sdk
+ENV ANDROID_HOME /builds/worker/android-sdk
+ENV ANDROID_SDK_HOME /builds/worker/android-sdk
 ENV PATH ${PATH}:${ANDROID_SDK_HOME}/tools:${ANDROID_SDK_HOME}/tools/bin:${ANDROID_SDK_HOME}/platform-tools:/opt/tools:${ANDROID_SDK_HOME}/build-tools/${ANDROID_BUILD_TOOLS}
 
 RUN curl -sfSL --retry 5 --retry-delay 10 https://dl.google.com/android/repository/sdk-tools-linux-${ANDROID_SDK_VERSION}.zip > sdk.zip \
     && unzip -q sdk.zip -d ${ANDROID_SDK_HOME} \
     && rm sdk.zip \
-    && mkdir -p /build/android-sdk/.android/ \
-    && touch /build/android-sdk/.android/repositories.cfg \
+    && mkdir -p /builds/worker/android-sdk/.android/ \
+    && touch /builds/worker/android-sdk/.android/repositories.cfg \
     && yes | sdkmanager --licenses \
     && sdkmanager --verbose "platform-tools" \
         "platforms;android-${ANDROID_PLATFORM_VERSION}" \
         "build-tools;${ANDROID_BUILD_TOOLS}" \
         "extras;android;m2repository" \
         "extras;google;m2repository"
 
+RUN chown -R worker:worker /builds/worker/android-sdk
+
 # Android NDK
 
-ENV ANDROID_NDK_VERSION "r19c"
+ENV ANDROID_NDK_VERSION "r20"
```

Comment:
Should we document somewhere that this is one location to update if we'll end up bumping the NDK version? (can happen in a separate PR)

Reply:
We should. I'll take care of that

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1632440